src/quadpype/hosts/blender/plugins/publish/test

```diff
@@ -0,0 +1,272 @@
+INFO: Collected project context
```

Comment:
Is a file used for testing in the futur or a reliquat of a previous test?
Since there is email address of Wizz user in the file I assume this is a reliquat that should be deleted